---
#Soft trunk parameters are defined here
robot name: 2segment
#######################
# PHYSICAL PARAMETERS #
#######################
#Physical segments of the arm
num segments: 2
#PCC sections per segment
sections per segment: 1
#Masses of segments and connectors, starting at topmost segment. Unit: kg
masses: [0.160, 0.020, 0.082, 0.023]
#Lengths of segments and connectors, starting at topmost segment. Unit: m
lengths: [0.125, 0.02, 0.125, 0.02]
#Diameters of segment top and bottoms, contains num_seg+1 values. Unit: m
diameters: [0.035, 0.028, 0.0198]
#Angle of arm relative to upright position. Unit: deg
armAngle: 180
#Shear moduli of the segments. Unit: Pa
shear modulus: [35000,62000]
#Drag coefficients of the segments
drag coef: [28000,8000]
#Maps Valve numbers to robot actuators, can be obtained with characterization script
valvemap: [3,2,1,0,4,5,6]

#######################
# MODEL CONFIGURATION #
#######################
#speed at which model self-updates, in hz
model update rate: 100
#model, valid args: augmented, lagrange
model type: "augmented"
#########################
# SENSOR CONFIGURATIONS #
#########################
#Sensors to be used, valid args: qualisys, bendlabs
sensors: [qualisys]
#sensor refresh rate, will be cut off to max values: qualisys 500hz, bendlabs 100hz
sensor refresh rate: 100
#bendlabs serial address
bendlabs address: /dev/ttyACM0


#########################
# AUTO CHARACTERIZATION #
#########################
#this section is dedicated to things which can only be obtained with scripts
#"real" actuation matrix which considers production errors
actuationMatrix: [1, 1, 1, 1, 1, 1, 1]

#polynomial coefficients which describe angular offset of the arm
angOffsetCoeffs: [1, 1, 1, 1]
...
